tests/foreman/ui/test_dashboard.py

```diff
@@ -187,16 +187,15 @@ def test_positive_task_status(session):
         })
         tasks = session.task.read_all()
         assert tasks['searchbox'] == 'state=stopped&result=warning'
-        assert (
-            tasks['table'][0]['Action'] ==
-            "Synchronize repository '{}'; product '{}'; "
-            "organization '{}'".format(repo.name, product.name, org.name))
+        task_name = "Synchronize repository '{}'; product '{}'; "\
+            "organization '{}'".format(repo.name, product.name, org.name)
```

Comment:
For strings that do not fit single line, please use brackets instead of backslash:
```python
        task_name = ("Synchronize repository '{}'; product '{}'; "
            "organization '{}'").format(repo.name, product.name, org.name)
```